### PR TITLE
Added redirect to remove index.php from Mission Portal's URL

### PR DESCRIPTION
**Pull request by aleksandrychev**
ChangeLog: Title

Ticket: ENT-6464

### CODE REVIEW

**Comment by aleksandrychev** [comment]
@nickanderson, please review

File: cfe_internal/enterprise/mission_portal.cf [hunk 1]
```diff
@@ -55,6 +55,15 @@ bundle agent cfe_internal_enterprise_mission_portal_apache
   reports:
     DEBUG|DEBUG_cfe_internal_enterprise_mission_portal_apache::
       "DEBUG $(this.bundle): Should stage '$(staged_config)' from '$(template)'.";
+
+  classes:
+
+    (policy_server|am_policy_hub).enterprise_edition.active_hub::
```
Comment (nickanderson) [inline cfe_internal/enterprise/mission_portal.cf:61]: mmmm active_hub may only be set on the hub if its running in HA AND if its not the slave. 

So, if you want this to apply on a hub that is not necessarily in ha I would drop the active_hub class, or say .!passive_ha_hub to say any hub that isn't passive.

**Review by nickanderson — CHANGES_REQUESTED** [review]
Looks OK, but i think the conditon for editing the config needs to be adjusted.

File: cfe_internal/enterprise/mission_portal.cf (hunk 1 above)
Comment (aleksandrychev) [reply cfe_internal/enterprise/mission_portal.cf:61]: ah, I thought active_hub is one more validation that hub is OK =) I will remove that

Comment (aleksandrychev) [reply cfe_internal/enterprise/mission_portal.cf]: @nickanderson done

**Review by nickanderson — APPROVED** [review]

**Pull request merged**